script/CommonDeployer.s.sol

```diff
@@ -37,18 +37,28 @@ struct CommonReport {
 }
 
 contract CommonActionBatcher {
-    // @dev lock to ensure we forbid calling engage/revoke methods after deployment
-    bool isLock;
```

Comment:
Lock is no longer needed, because with address deployer we can already control the locking mechanism